fs/communitarian-discourse-plugin
# 174274064 community resolutions

#### A brief description of the pull request.

[PT Story 174274064](https://www.pivotaltracker.com/story/show/174274064)

[Design](https://projects.invisionapp.com/d/main#/console/20307299/426705102/preview)

#### Screenshots in case of UI changes

<img width="887" alt="Снимок экрана 2020-08-26 в 18 07 08" src="https://user-images.githubusercontent.com/46020998/91321177-06fd6780-e7c7-11ea-8511-e7b7e6a446fc.png">

# Test plan

#### List of steps to manually test introduced functionality:

- Go to Community page ([Staff like](http://localhost:3000/c/staff/3))
- Check if everything is working correctly
